src/Bridge/Symfony/Bundle/Test/ApiTestAssertionsTrait.php

```diff
@@ -91,18 +95,34 @@ public static function assertJsonEquals($json, string $message = ''): void
     public static function assertArraySubset($subset, $array, bool $checkForObjectIdentity = false, string $message = ''): void
     {
         $constraint = new ArraySubset($subset, $checkForObjectIdentity);
+
         static::assertThat($array, $constraint, $message);
     }
 
     /**
-     * @param array|string $jsonSchema
+     * @param Schema|array|string $jsonSchema
```

Comment:
Could be `\ArrayObject` instead of `Schema` isn't it?

Reply:
Actually, do we really need to support this here?